### PR TITLE
Fix #46 - Check that timestamps are included in all feed elements

### LINKED ISSUE #46: Check that timestamps are included in all feed elements
Header, TripUpdates, VehiclePositions, and ServiceAlerts.  This would generate a warning.  See https://groups.google.com/d/msg/gtfs-realtime/qNc7ButQbW8/tDwanvClBQAJ.

### PR DESCRIPTION
**Pull request by mohangandhiGH**
This rule is Implemented as warning "w001"

Previously timestamp validation was checked only in FeedHeader. Now checked in TripUpdate and VehiclePositions too.

Added Rules_Implemented.md file to keep track of rules implemented. For now, added warnings and rules that are implemented previously. Will update this file as we implement new warnings and rules.

### CODE REVIEW

File: src/main/java/edu/usf/cutr/gtfsrtvalidator/validation/entity/TimestampValidation.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2011 Nipuna Gunathilake.
```
Comment (barbeau) [inline src/main/java/edu/usf/cutr/gtfsrtvalidator/validation/entity/TimestampValidation.java]: Can you please update the copyright header to  "Copyright (C)  2011-2016  Nipuna Gun..., University of South Florida"?

**Comment by barbeau** [comment]
Also, could you check out why the Travis builds are failing?

File: src/main/java/edu/usf/cutr/gtfsrtvalidator/validation/entity/TimestampValidation.java (hunk 1 above)
Comment (barbeau) [inline src/main/java/edu/usf/cutr/gtfsrtvalidator/validation/entity/TimestampValidation.java]: Sorry, that should be 2017, not 2016.

**Comment by mohangandhiGH** [comment]
@barbeau included the changes requested in the commit :)

Travis Build failed due to  #52  was not merged before. Build passed after the merge.

**Comment by barbeau** [comment]
Excellent, thanks, looks good!

**Pull request merged**